### ISSUE TITLE
Failed to load the minimap-autohider package

### ISSUE THREAD

**Issue by tesseslol** [ansballard/minimap-autohider#2]
[Enter steps to reproduce:]

1. Open Atom in a project

**Atom**: 1.14.4 ia32
**Electron**: 1.3.13
**OS**: Unknown Windows version
**Thrown From**: [minimap-autohider](https://github.com/ansballard/minimap-autohider) package 1.3.1

### Stack Trace

Failed to load the minimap-autohider package

```
At variable @TransitionDuration is undefined in C:\Users\Tessari Davide\.atom\packages\minimap-autohider\styles\minimap-autohider.less:14:26

LessError: variable @TransitionDuration is undefined
  at /packages/minimap-autohider/styles/minimap-autohider.less:14:26
```

### Commands

```
     -0:13.9.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
angular-2-typeScript-snippets 0.6.3 
angularjs 0.4.0 
api-docs 0.0.6 
atom-alignment 0.12.1 
atom-beautify 0.29.17 
atom-clock 0.1.5 
atom-pair 2.0.10 
atom-ternjs 0.17.2 
atom-typescript 10.1.14 
atomic-chrome 0.3.0 
autoclose-html 0.23.0 
autocomplete-json 5.3.0 
autocomplete-paths 1.0.5 
autocomplete-robot-framework 3.1.0 
bezier-curve-editor 0.7.2 
browser-plus 0.0.87 
build 0.67.0 
busy 0.7.0 
caniuse 0.9.0 
change-case 0.6.3 
chary-tree-view 0.2.3 
clipboard-plus 0.5.1 
code-peek 1.4.16 
color-picker 2.2.5 
console-log 1.10.0 
css-snippets 1.1.0 
css-spy 0.6.0 
disable-keybindings 0.6.0 
docblockr 0.9.1 
double-tag 0.4.0 
editorconfig 2.2.2 
emmet 2.4.3 
expose 0.14.0 
file-icons 2.0.16 
git-diff-details 1.4.0 
git-log 0.4.1 
git-plus 7.3.3 
git-time-machine 1.5.6 
gitignore-snippets 0.2.5 
haskell-grammar 0.4.0 
hey-pane 0.2.0 
highlight-selected 0.12.0 
html-to-css 0.2.2 
imdone-atom 2.1.19 
imdone-atom-github 0.1.11 
javascript-snippets 1.2.1 
jumpy 3.1.3 
language-babel 2.55.6 
language-gitignore 0.3.0 
language-graphql 0.8.0 
language-ini 1.18.0 
language-javascript-jsx 0.3.7 
language-lua 0.9.11 
language-ocaml 1.1.2 
language-robot-framework 2.2.0 
language-swift 0.5.0 
language-thrift 1.0.2 
less-than-slash 0.16.0 
linter 1.11.23 
linter-jshint 3.0.2 
linter-write-good 0.8.0 
MagicPython 1.0.7 
markdown-preview-plus 2.4.8 
markdown-writer 2.6.4 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-autohider 1.3.1 
minimap-bookmarks 0.4.2 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.1 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-pigments 0.2.2 
minimap-selection 4.4.0 
monokai 0.22.0 
multi-cursor-plus 1.2.0 
nuclide 0.208.0 
nuclide-format-js 0.0.36 
pdf-view 0.55.0 
pigments 0.39.0 
pinned-tabs 0.4.2 
platformio-ide-terminal 2.3.1 
process-palette 0.12.0 
project-manager 3.3.3 
project-sidebar 1.3.0 
qolor 0.4.1 
quick-highlight 0.8.1 
refactor 0.11.3 
regex-railroad-diagram 0.16.0 
remember-file-positions 0.2.2 
robot-selenium-snippet 1.0.1 
script 3.14.1 
set-syntax 0.3.2 
seti-ui 1.6.1 
simple-drag-drop-text 0.3.4 
snippet-generator 0.3.2 
sort-lines 0.14.0 
sublime-block-comment 0.5.1 
Sublime-Style-Column-Selection 1.7.3 
symbols-tree-view 0.13.2 
sync-settings 0.8.1 
synced-sidebar 0.4.3 
tablr 1.8.1 
tabs-to-spaces 1.0.3 
todo-show 1.11.0 
toggle-quotes 1.0.0 
tomatimer 1.2.3 
tool-bar 1.0.1 
tree-ignore 0.6.2 
tree-view-autoresize 1.5.0 
tree-view-breadcrumb 0.8.1 
tree-view-git-branch 0.1.2 
tree-view-git-status 1.3.1 
tree-view-search-bar 0.1.3 
vim-mode-clipboard-plus 0.2.0 
vim-mode-plus 0.82.3 
vim-mode-plus-ex-mode 0.9.0 
zentabs 0.8.8 
```

**Comment by ansballard** [comment]
That's weird, the variable is definitely there initially. There's probably some bad logic with how `custom.less` is getting generated from the package settings. I'll try to get it sorted out tonight.

Ew, looks like something in the 1.14.4 update broke it. I guess that's what I get for hacking in configurable styles ¯\\_(ツ)_/¯

Pushed a quick patch that disables the "Transition Duration" setting so the plugin at least loads. After updating to `1.3.2` there shouldn't be any warnings, only change is that your minimap will slide out in 100ms, regardless of what's in your settings.

**Comment by ansballard** [comment]
Just pushed `1.3.3`, which should fix the issue. I'm still not really sure what was causing the less variable not to be initialized after an update. I'm just going to go ahead and assume it was a race condition, which I've fixed by adding a `default.less` file that initializes the variable. That way, the variable will always be set to _something_, which should avoid that specific crash. Until atom adds some kind of real connection to the settings page from `less`, this will continue to be super hacky. Closing for now, let me know if the issue persists after updating.

Thanks for the report @tesseslol!
